media-service/src/main/java/org/arpha/exception/handler/MediaExceptionHandler.java

```diff
@@ -0,0 +1,30 @@
+package org.arpha.exception.handler;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.arpha.exception.FileNotFoundException;
+import org.arpha.exception.FileUploadException;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ProblemDetail;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@Slf4j
+@RestControllerAdvice
+@Order(2)
```

Comment:
Consider making it just OrderedHIGHEST_PRECEDENCE.

Reply:
forgot